src/Compilers/Core/CodeAnalysisTest/EditAndContinue/SemanticEditDescription.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.Emit;
+
+namespace Microsoft.CodeAnalysis.EditAndContinue.UnitTests;
+
+internal sealed class SemanticEditDescription(
```

Comment:
Should these new types go in CodeAnalysis.Test.Utilities.csproj?

Reply:
Maybe. Our test utilities projects need refactoring. Out of scope for this change.